docs/user_guide/configuring.rst

```diff
@@ -121,6 +121,47 @@ an external site. You can add external links to the nav bar like so:
      ]
    }
 
+Adding favicons
+===============
+
+``pydata_sphinx_theme`` supports the
+`standard sphinx favicon configuration <https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_favicon>`_,
+using ``html_favicon``.
+
+Additionally, ``pydata_sphinx_theme`` allows you to add any number of
+browser- or device-specific favicons of any size. To define arbitrary favicons,
+use the ``favicons`` configuration key:
+
+.. code-block:: python
+
+   html_theme_options = {
+      "favicons": [
+         {
+            "rel": "icon",
+            "sizes": "16x16",
+            "href": "https://secure.example.com/favicon/favicon-16x16.png",
```

Comment:
Could this also be a local file relative to `_static` ? (like you can do with the default favicon)

Reply:
Good idea - `_static` paths are now supported as well!